src/lib/application-state-stores/explorer-stores.ts

```diff
@@ -0,0 +1,81 @@
+import { writable } from "svelte/store";
+import type { MetricsExplorerEntity } from "$lib/redux-store/explore/explore-slice";
+import type { MetricViewMetaResponse } from "$common/rill-developer-service/MetricViewActions";
+import type { TimeSeriesTimeRange } from "$common/database-service/DatabaseTimeSeriesActions";
+
+export interface MetricsExplorerStoreType {
+  entities: Record<string, MetricsExplorerEntity>;
```

Comment:
Elsewhere in the application we use Contexts to pass entityID-specific stores around, e.g.: https://github.com/rilldata/rill-developer/blob/a2a8e052ac05210da26ef6856b13512f1b2c611e/src/routes/__layout.svelte#L36

Do you like the dictionary-based approach (`Record<string, MetricssExplorerEntity>`) more than the context approach? I googled around a bit for best practices, but couldn't find a consensus. Happy to go with this approach if you prefer it – I just want to be conscious about deviating from our other Context-based methodology.

Reply:
Hmmm not sure what the context gives us over just import/export of the store. I feel this is simpler.